Description: Don't build vfat related code on non-linux platforms
Author: Petr Salinger <Petr.Salinger@seznam.cz>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=576938
Index: tracker/src/miners/fs/tracker-miner-files.c
===================================================================
--- tracker.orig/src/miners/fs/tracker-miner-files.c	2010-04-01 17:24:01.000000000 +0200
+++ tracker/src/miners/fs/tracker-miner-files.c	2010-04-08 21:03:44.183099610 +0200
@@ -21,8 +21,10 @@
 
 #include <sys/statvfs.h>
 #include <fcntl.h>
+#ifdef __linux__
 #include <sys/ioctl.h>
 #include <linux/msdos_fs.h>
+#endif /* __linux__ */
 #include <unistd.h>
 
 #include <glib/gi18n.h>
@@ -1777,6 +1779,7 @@
 	 */
 	is_hidden = file_info && g_file_info_get_is_hidden (file_info);
 
+#ifdef __linux__
 	/* Second we check if the file is on FAT and if the hidden
 	 * attribute is set. GIO does this but ONLY on a Windows OS,
 	 * not for Windows files under a Linux OS, so we have to check
@@ -1796,6 +1799,7 @@
 			close (fd);
 		}
 	}
+#endif /* __linux__ */
 
 	if (is_hidden) {
 		/* FIXME: We need to check if the file is actually a
